content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md

```diff
@@ -131,7 +131,7 @@ To enable {% data variables.product.prodname_actions %} on {% data variables.pro
 {%- ifversion actions-ghes-gcp-storage %}
 * Google Cloud Storage
 {%- endif %}
-* S3-compatible MinIO Gateway for NAS
+* S3-compatible MinIO cluster
```

Comment:
Shouldn't it just be "MinIO storage"? We should be consistent with the product name used here.

Reply:
MinIO Server is the replacement for MinIO Gateway. Its a software-defined storage solution.